.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+install:
+- npm install
+- npm run bower:install
```

Comment:
nope, you don't need to. see https://github.com/policycompass/policycompass-frontend/pull/464